packages/react-native-platform-override/src/GitReactFileRepository.ts

```diff
@@ -235,13 +235,16 @@ export default class GitReactFileRepository
       throw new Error(`${reactNativeVersion} is not a valid semver version`);
     }
 
-    // Stable releases of React Native use a tag where nightly releases embed
-    // a commit hash into the prerelease tag of 0.0.0 versions
+    // Nightly builds are in the form of either 0.0.0-<commitHash> or
+    // 0.0.0-<commitHash>-<date>-<time>.
```

Comment:
Hey Nick, since the 0.0.0 is not increasing, any feed we publish this package too will get confused about the latest package, since it assumes that the prerelease will still be monotonically increasing... Should we use:
`0.0.0-<yyyymmdd>-<HH:mm>-<comitHash>`

Reply:
Ah, sorry I didn't realize this is about FB's react-native versioning, since they already are using the commitHash, there is no change.